data/i18n/hi/hi.toml

```diff
@@ -1,180 +1,180 @@
 
 [version_check_mustbeorlater]
-other = "Your Kubernetes server must be at or later than version "
+other = "आपका कुबेरनेट्स सर्वर पर या बाद में संस्करण की तुलना में होना चाहिए "
 
 [version_check_tocheck]
-other = "To check the version, enter "
+other = "संस्करण की जांच करने के लिए, दर्ज करें "
 
 [caution]
-other = "Caution:"
+other = "सावधानी:"
 
 [note]
-other = "Note:"
+other = "टिप्पणी:"
 
 [warning]
-other = "Warning:"
+other = "चेतावनी:"
 
 [main_read_about]
-other = "Read about"
+other = "के बारे में पढ़ा"
 
 [main_read_more]
-other = "Read more"
+other = "अधिक पढ़ें"
 
 [main_github_invite]
-other = "Interested in hacking on the core Kubernetes code base?"
+other = "कोर कुबेरनेट्स कोड बेस पर हैकिंग के इच्छुक हैं?"
 
 [main_github_view_on]
-other = "View On GitHub"
+other = "गिटहब पर देखें"
 
 [main_github_create_an_issue]
-other = "Create an Issue"
+other = "एक इशू बनाएं"
 
 [main_community_explore]
-other = "Explore the community"
+other = "समुदाय का अन्वेषण करें"
 
 [main_kubernetes_features]
-other = "Kubernetes Features"
+other = "कुबेरनेट्स विशेषताएं"
 
 [main_cncf_project]
-other = """We are a <a href="https://cncf.io/">CNCF</a> graduated project</p>"""
+other = """हम एक <a href="https://cncf.io/">CNCF</a> स्नातक परियोजना</p> हैं"""
 
 [main_kubeweekly_baseline]
-other = "Interested in receiving the latest Kubernetes news? Sign up for KubeWeekly."
+other = "नवीनतम कुबेरनेट्स समाचार प्राप्त करने के इच्छुक हैं? KubeWeekly के लिए साइन अप करें।"
 
 [main_kubernetes_past_link]
-other = "View past newsletters"
+other = "पिछले न्यूज़लेटर देखें"
 
 [main_kubeweekly_signup]
-other = "Subscribe"
+other = "सब्सक्राइब"
 
 [main_contribute]
-other = "Contribute"
+other = "योगदान"
 
 [main_edit_this_page]
-other = "Edit This Page"
+other = "इस पृष्ठ को संपादित करें"
 
 [main_page_history]
-other ="Page History"
+other ="पृष्ठ इतिहास"
 
 [main_page_last_modified_on]
-other = "Page last modified on"
+other = "पृष्ठ अंतिम बार संशोधित किया गया"
 
 [main_by]
-other = "by"
+other = "द्वारा"
 
 [main_documentation_license]
-other = """The Kubernetes Authors | Documentation Distributed under <a href="https://git.k8s.io/website/LICENSE" class="light-text">CC BY 4.0</a>"""
+other = """कुबेरनेट्स लेखक | <a href="https://git.k8s.io/website/LICENSE" class="light-text">CC BY 4.0</a> के अंतर्गत वितरित दस्तावेज़"""
 
 [main_copyright_notice]
-other = """The Linux Foundation &reg;. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our <a href="https://www.linuxfoundation.org/trademark-usage" class="light-text">Trademark Usage page</a>"""
+other = """Linux फाउंडेशन &reg;। सभी अधिकार सुरक्षित। Linux फाउंडेशन ने ट्रेडमार्क पंजीकृत किया है और ट्रेडमार्क का उपयोग करता है। Linux फाउंडेशन के ट्रेडमार्क की सूची के लिए, कृपया हमारा <a href="https://www.linuxfoundation.org/trademark-usage" class="light-text">ट्रेडमार्क उपयोग पृष्ठ</a> देखें।"""
```

Comment:
```sugestion
other = """The Linux Foundation &reg;। सभी अधिकार सुरक्षित। The Linux Foundation ने ट्रेडमार्क पंजीकृत किया है और ट्रेडमार्क का उपयोग करता है। Linux फाउंडेशन के ट्रेडमार्क की सूची के लिए, कृपया हमारा <a href="https://www.linuxfoundation.org/trademark-usage" class="light-text">ट्रेडमार्क उपयोग पृष्ठ</a> देखें।"""
```
?

Reply:
other = """The Linux Foundation &reg;। सभी अधिकार सुरक्षित। The Linux Foundation ने ट्रेडमार्क पंजीकृत किया है और ट्रेडमार्क का उपयोग करता है। Linux `Foundation` के ट्रेडमार्क की सूची के लिए, कृपया हमारा <a href="https://www.linuxfoundation.org/trademark-usage" class="light-text">ट्रेडमार्क उपयोग पृष्ठ</a> देखें।"""


can we change फाउंडेशन to Foundation (hightlighted).

Reply:
Yes, we should

Reply:
The name of the organization is “Linux Foundation” so we should either transcribe both “Linux” and “Foundation” into dévanāgarī, or we should use the name in the Roman alphabet.

Reply:
I updated it as Linux Foundation. I think it is fine to do so.